subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -605,6 +606,9 @@ u8_t ll_adv_enable(u8_t enable)
 		conn_lll->slave.window_widening_event_us = 0;
 		conn_lll->slave.window_size_prepare_us = 0;
 		/* FIXME: END: Move to ULL? */
+#if defined(CONFIG_BT_CTLR_CONN_PROP_META)
+		memset(&conn_lll->prop_meta, 0, sizeof(conn_lll->prop_meta));
```

Comment:
Maybe this should call a function to initialize the meta data instead of assuming that zero-initialization is the right thing to do.

Reply:
> Maybe this should call a function to initialize the meta data instead of assuming that zero-initialization is the right thing to do.

I would prefer to avoid having vendor specific meta property functions as well. You can always structure your meta data such that setting all to '0' will be an unambiguous operation.